### ISSUE TITLE
HOW to run a Demo?

### ISSUE THREAD

**Issue by Helen-Cheung** [engcang/TROT-Q#1]
I am a beginner in ROS. May I ask why the quadruped robot does not move when I download the code and compile it according to the dependent environment? I found that the Scripts file in TROT-Q doesn't seem to be running.

**Comment by engcang** [comment]
Hi @Helen-Cheung

If you look into [this](https://github.com/engcang/TROT-Q#how-to-run) section of readme, 
You should move Jackal mobile robot manually to escape, then the robot automatically tracks it.

[This](https://github.com/engcang/TROT-Q/blob/main/TROT-Q/launch/gazebo.launch#L24-L27) block of launch file should be changed to use your joystick.
If you do not have joystick. try to move jackal robot publishing /jackal/jackal_velocity_controller/cmd_vel topic in geometry_msgs/Twist type, as [this](https://github.com/engcang/TROT-Q/blob/main/jackal_package_for_gazebo/scripts/joy_controller.py#L27-L28) does.

**Comment by Helen-Cheung** [comment]
Hi @engcang  
Thank you for your quick reply! I have used Joystick to control Jackal. But the situation is that only Jackal is moving, and Anymal will not follow it. 
When I click `gazebo_ros_control` of 'anymal_b', a prompt appears on the command line.
`Topic [/data://world/default/model/anymal_b/plugin/gazebo_ros_control/joint_cmd] is not valid.`
If this is the cause of the problem?  
Another question is, should I uncomment MPC_node and FAST-LIO in the [main.launch](https://github.com/engcang/TROT-Q/blob/main/TROT-Q/launch/main.launch) file? I tried this, but still couldn't make Anymal move.
Thank you very much.

**Comment by engcang** [comment]
@Helen-Cheung 
Yeah, as you mentioned, those two lines should be uncommented in the main.launch file.

I just git cloned, built, and ran the code and it works properly.
If you launch both gazebo.launch and main.launch, do you see Rviz with OctoMap and object detection with YOLO? 
if you cannot, please verify every nodes in the launch files work properly, which means you should check every requirements are installed well.

**Comment by engcang** [comment]
@Helen-Cheung Any updates?

**Comment by chenweigu** [comment]
Excuse me, may I ask a question? after running "gazebo. launch", in the "rostopic list", only "/os_ cloud_ node/points" without "/os_ cloud_ node/imu", anymal_b itself has a camera and IMU. The blogger adds an ouster lidar and IMU in the anymal_lidar_imu.urdf. Why is there no corresponding topic of ouster IMU? After running "main.launch", the laserMapping node does not published /tf...

**Comment by chenweigu** [comment]
May I ask if you only used point information and not imu information when using fastlio? Because after running 'gazebo. launch', I only found '/os_cloud node/points' in the' rostopic list ', but after running' main. launch ', the' lasermapping' node did not publish the' tf 'topic, and rviz reported error.

after running main.launch, rosopic info /os_cloud_node_imu:
Type: sensor_msgs/Imu

Publishers: None

Subscribers: 
 * /laserMapping (http://wgchen:36975/)
 

rosnode info /lasermapping:
Node [/laserMapping]
Publications: 
 * /Laser_map [sensor_msgs/PointCloud2]
 * /Odometry [nav_msgs/Odometry]
 * /cloud_effected [sensor_msgs/PointCloud2]
 * /cloud_registered [sensor_msgs/PointCloud2]
 * /cloud_registered_body [sensor_msgs/PointCloud2]
 * /path [nav_msgs/Path]
 * /rosout [rosgraph_msgs/Log]

Subscriptions: 
 * /clock [rosgraph_msgs/Clock]
 * /os_cloud_node/imu [unknown type]
 * /os_cloud_node/points [sensor_msgs/PointCloud2]

Services: 
 * /laserMapping/get_loggers
 * /laserMapping/set_logger_level


contacting node http://wgchen:36975/ ...
Pid: 20518
Connections:
 * topic: /rosout
    * to: /rosout
    * direction: outbound (58865 - 127.0.0.1:60546) [10]
    * transport: TCPROS
 * topic: /Odometry
    * to: /tf_broadcaster
    * direction: outbound (58865 - 127.0.0.1:60558) [15]
    * transport: TCPROS
 * topic: /Odometry
    * to: /mpc_controller
    * direction: outbound (58865 - 127.0.0.1:60566) [18]
    * transport: TCPROS
 * topic: /path
    * to: /rviz
    * direction: outbound (58865 - 127.0.0.1:60582) [20]
    * transport: TCPROS
 * topic: /clock
    * to: /gazebo (http://wgchen:44673/)
    * direction: inbound (50198 - wgchen:48847) [17]
    * transport: TCPROS
 * topic: /os_cloud_node/points
    * to: /gazebo (http://wgchen:44673/)
    * direction: inbound (50254 - wgchen:48847) [16]
    * transport: TCPROS
